src/chttpd/src/chttpd.erl

```diff
@@ -268,6 +275,11 @@ handle_request_int(MochiReq) ->
             span_ok(HttpResp),
             {ok, Resp};
         #httpd_resp{status = aborted, reason = Reason} ->
+            ?LOG_ERROR(#{
+                what => abnormal_response_termation,
+                id => get(nonce),
```

Comment:
1) I think we can use `logger:set_process_metadata(#{http_request_id => Nonce})` once on the line after we calculated `Nonce` and do not pass it explicitly everywhere. 

2) I think `id` name is too generic this would make correlation of log events on the external log querying system problematic.

Reply:
Excellent idea on `set_process_metadata`, I imagine there are a few locations where that could come in handy over time.

I'm OK changing `id` as well.

Reply:
Addressed in 5a139b249